### ISSUE TITLE
State Classes are not applied to widgets that have structural CSS classes applied

### ISSUE THREAD

**Issue by agubler** [dojo/widgets#73]
**Bug**

With the initial implementation of structural CSS, classes that have been configured with structural CSS will not ever have classes applied via the widgets `state`

I think this will probably be addressed by #69 but in the meantime the changes will break consumers (i.e. todoMVC (verified)/MonsterApp (not verified)).

Package Version: master

```js
import createMemoryStore from 'dojo-stores/createMemoryStore';
import createButton from '../createButton';
import { createProjector } from '../projector';

const widgetStore = createMemoryStore({
	data: [
		{ id: 'id', classes: [ 'fred' ], label: 'label' }
	]
});

const button = createButton({
	id: 'id',
	stateFrom: widgetStore
});

const panelProjector = createProjector({ root: document.body });
panelProjector.append(button);
panelProjector.attach();
```

**Expected**

Expected the `button` to have class `fred` applied.

**Actual**

Only the structural css is applied to the `button` widget.

**Comment by kitsonk** [comment]
Well, #69 needs to address this.  Right now it just doesn't work one way or the other.  If we could resolve #69 quickly, we should be able to maintain backwards compatibility.

**Comment by agubler** [comment]
Sure, as I said I believe #69 should address - just making a record of the exact issue.

**Comment by agubler** [comment]
Resolved by #74
